examples/pom.xml

```diff
@@ -25,16 +25,27 @@
             <artifactId>slf4j-simple</artifactId>
             <version>2.0.7</version>
         </dependency>
+
```

Comment:
`mvn clean install` under examples folder seems broken. could you please fix it before merging?

Reply:
same here. getting the below error 
```
ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /Users/ntantry/dev/intuit_projects/numaflow-java/examples/src/main/java/io/numaproj/numaflow/examples/reducesession/counter/CountFactory.java:[22,15] cannot find symbol
  symbol:   method awaitTermination()
  location: variable server of type io.numaproj.numaflow.sessionreducer.Server
[ERROR] /Users/ntantry/dev/intuit_projects/numaflow-java/examples/src/main/java/io/numaproj/numaflow/examples/sideinput/simple/SimpleSideInput.java:[37,15] cannot find symbol
  symbol:   method awaitTermination()
  location: variable server of type io.numaproj.numaflow.sideinput.Server
[ERROR] /Users/ntantry/dev/intuit_projects/numaflow-java/examples/src/main/java/io/numaproj/numaflow/examples/reduce/sum/SumFactory.java:[18,15] cannot find symbol
  symbol:   method awaitTermination()
  location: variable server of type io.numaproj.numaflow.reducer.Server
[ERROR] /Users/ntantry/dev/intuit_projects/numaflow-java/examples/src/main/java/io/numaproj/numaflow/examples/reduce/count/CounterFactory.java:[32,15] cannot find symbol
  symbol:   method awaitTermination()
  location: variable server of type io.numaproj.numaflow.reducer.Server
[ERROR] /Users/ntantry/dev/intuit_projects/numaflow-java/examples/src/main/java/io/numaproj/numaflow/examples/sourcetransformer/eventtimefilter/EventTimeFilterFunction.java:[32,15] cannot find symbol
  symbol:   method awaitTermination()
  location: variable server of type io.numaproj.numaflow.sourcetransformer.Server
[ERROR] /Users/ntantry/dev/intuit_projects/numaflow-java/examples/src/main/java/io/numaproj/numaflow/examples/mapstream/flatmapstream/FlatMapStreamFunction.java:[26,15] cannot find symbol
  symbol:   method awaitTermination()
  location: variable server of type io.numaproj.numaflow.mapstreamer.Server
[ERROR] /Users/ntantry/dev/intuit_projects/numaflow-java/examples/src/main/java/io/numaproj/numaflow/examples/source/simple/SimpleSource.java:[37,15] cannot find symbol
  symbol:   method awaitTermination()
  location: variable server of type io.numaproj.numaflow.sourcer.Server
[ERROR] /Users/ntantry/dev/intuit_projects/numaflow-java/examples/src/main/java/io/numaproj/numaflow/examples/sink/simple/SimpleSink.java:[26,15] cannot find symbol
  symbol:   method awaitTermination()
  location: variable server of type io.numaproj.numaflow.sinker.Server
[ERROR] /Users/ntantry/dev/intuit_projects/numaflow-java/examples/src/main/java/io/numaproj/numaflow/examples/map/forward/ForwardFunction.java:[21,15] cannot find symbol
  symbol:   method awaitTermination()
  location: variable server of type io.numaproj.numaflow.mapper.Server
[ERROR] /Users/ntantry/dev/intuit_projects/numaflow-java/examples/src/main/java/io/numaproj/numaflow/examples/reducestreamer/sum/SumFactory.java:[22,15] cannot find symbol
  symbol:   method awaitTermination()
  location: variable server of type io.numaproj.numaflow.reducestreamer.Server
[ERROR] /Users/ntantry/dev/intuit_projects/numaflow-java/examples/src/main/java/io/numaproj/numaflow/examples/sideinput/udf/SimpleMapWithSideInput.java:[47,15] cannot find symbol
  symbol:   method awaitTermination()
  location: variable server of type io.numaproj.numaflow.mapper.Server
[ERROR] /Users/ntantry/dev/intuit_projects/numaflow-java/examples/src/main/java/io/numaproj/numaflow/examples/map/flatmap/FlatMapFunction.java:[25,15] cannot find symbol
  symbol:   method awaitTermination()
  location: variable server of type io.numaproj.numaflow.mapper.Server

```

Reply:
pls ignore my above comment. Running `mvn clean install` in the parent folder, got rid of this error message. Now I see the same error message as @KeranYang

Reply:
I have fixed it. Please let me know if you still see the error.

Reply:
working fine now.